index.html

```diff
@@ -1,16 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
-<body>
-    <!-- Add your markup here -->
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Responsive Cake webpage</title>
+        <link rel="stylesheet" href="css/style.css"
+    </head>
 
+    <body>
+        <header class="header">
+            <div class="logo-contact">
+                <span class="logo">
+                    <img src="https://lh3.googleusercontent.com/proxy/BhNZ79ahYgRQMqiAm86fWvnJ0jSJWWU_18X2V1cfHXopFFu2ZixnwSJppz7HIq9zjwwrSXHJU_EA8qnklrT2JGHvh-IQK5A" alt="cake animated image"/>
+                </span>
+                <div class="contact">
+                    <span><a href="#" target="_blank">Contact</a></span>
+                </div>
+            </div>
+            <div class="header-nav-list">
+                <nav class="nav-bar">
+                    <ul> 
+                        <li><a href="#">Home</a></li>
+                        <li><a href="#">cakes</a></li>
+                        <li><a href="#">ordering</a></li>
+                        <li class="hide"><a href="#">lessons</a></li>
+                        <li class="hide"><a href="#">about</a></li>
```

Comment:
I would try to avoid hiding links and information to accommodate a responsive design as you are limiting the functionality of your site. In this instance you are depriving a mobile users of two pages, the about page would be particularly important for example as it is one of the first pages people want to read when they want to find out what you do as a company. 

In this instance I would instead have stacked the links on top of each other for mobile.

Reply:
I tried unsuccessfully to do the same as you suggested.